### PR TITLE
fixed units etc. plus sorted main ode 

### PR DESCRIPTION
**Pull request by J-Bow**

### CODE REVIEW

File: Our Scripts/EWvsZ.py [hunk 1]
```diff
@@ -24,4 +25,9 @@
...
 EW=[mod.EW(i) for i in z]
 
-plotter.plot(z,EW,"A plot of the equivalent width against redshift","Redshift","EW [Å]",path)
\ No newline at end of file
+x = pylab.array([2.5,2.8,2.9,3.1, 3.3, 3.7, 4.1, 4.5, 4.8, 5.0, 5.3])
+y = pylab.array([117.134349876, 122.113769531, 172.679885864, 143.5936203, 149.371124268, 96.3648490905, 189.002449036, 181.127731323, 95.0448436864, 125.935153962, 143.343048096])
+
+plt.scatter(x, y)
+plt.plot(z, EW )
+plt.show()
```
Comment (why-keith) [inline Our Scripts/EWvsZ.py]: Is this section meant to replace the master?

File: Our Scripts/MPhys_model.py [hunk 2]
```diff
@@ -60,8 +60,8 @@ def t_rec(z): #s recombination time
 def E_ion(z):#Hz/erg
     return 10**25.6 #10**(24.4 + math.log10(1 + z))
 
-def Cubic(x, a1, a2, a3, a4):
-    return a1*x**3 + a2*x**2 +a3*x +a4
+def squared(x, a1, a2, a3):
+    return a1*x**2 + a2*x +a3
```
Comment (why-keith) [inline Our Scripts/MPhys_model.py:85]: Is the squared function meant to replace the cubic function?

File: Our Scripts/MPhys_model.py [hunk 3]
```diff
@@ -102,19 +102,17 @@ def P_L_Lya(z): # luminosity density of lyman alpha
...
 
-    x = pylab.array([2.2,2.5,2.8,3.0,3.2, 3.3, 3.7, 4.1, 4.6, 4.8, 5.1, 5.3, 5.8,7,10,11,12,13,14,16])
-    y1 = pylab.array([0.52, 0.74, 0.77, 0.88, 0.84, 0.85, 1.01, 0.87, 1.19, 1.12, 1.27, 1.08, 1.10,.7,0.5,0.3,0.10,0.05,0.001,0.001])  # data from SC4K Sobral 
+    x = pylab.array([2.2,2.5,2.8,3.0,3.2, 3.3, 3.7, 4.1, 4.6, 4.8, 5.1, 5.3, 5.8])#,7,10,11,12,13,14,16])
+    y1 = pylab.array([ 0.52, 0.74, 0.77, 0.88, 0.84, 0.85, 1.01, 0.87, 1.19, 1.12, 1.27, 1.08, 1.10]) #,.7,0.5,0.3,0.10,0.05,0.001,0.001])  # data from SC4K Sobral 
     y = [math.log10(i*10**40) for i in y1]
-    y_sigma = pylab.array([0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01, 0.01, 0.01, 0.01 ,0.01,0.01,0.01, 0.01, 0.01, 0.01 ,0.01])
-    p2 = pylab.polyfit(x, y, 3.0)
+    y_sigma = pylab.array([0.05,0.075,0.095, 0.095,0.09,0.095,0.18,0.13,0.35,0.32,0.40, 0.185, 0.185])#,0.01,0.01,0.01, 0.01, 0.01, 0.01 ,0.01])
+    p2 = pylab.polyfit(x, y, 2.0)
```
Comment (why-keith) [inline Our Scripts/MPhys_model.py:130]: Different data sets here, which one is the most up to date?

File: Our Scripts/MPhys_model.py [hunk 4]
```diff
@@ -126,14 +124,18 @@ def EW(z):
...
+    y_err_down =pylab.array([53.82247, 53.55149, 90.78461, 72.25881, 74.65443, 39.08451, 96.45726, 85.27828, 42.09321, 52.08989, 71.00167, 186.34499])
+    error = [(y_err_up[i]-y_err_down[i])/2 for i in range(len(y_err_down))]
     p2 = np.polyfit(x, y, 1.0)
     p = pylab.poly1d(p2)
     Params = p.coefficients
-    pfit, pcov = curve_fit(linear, x, y, p0=Params, sigma=y_sigma)
+    pfit, pcov = curve_fit(linear, x, y, p0=Params, sigma=error)
     a1,a2= pfit[0], pfit[1]
+    aver = np.mean(y)
+    print(aver)
```
Comment (why-keith) [inline Our Scripts/MPhys_model.py:159]: Just double checking that this is more up to date

File: Our Scripts/EWvsZ.py (hunk 1 above)
Comment (J-Bow) [reply Our Scripts/EWvsZ.py]: yeah, it means we can plot data points on top of the graph

File: Our Scripts/MPhys_model.py (hunk 2 above)
Comment (J-Bow) [reply Our Scripts/MPhys_model.py:85]: yeah, we decided they were squares not cubics

File: Our Scripts/MPhys_model.py (hunk 3 above)
Comment (J-Bow) [reply Our Scripts/MPhys_model.py:130]: the y_sigma with values other than 0.01 is the correct one (it was just a place holder)

File: Our Scripts/MPhys_model.py (hunk 4 above)
Comment (J-Bow) [reply Our Scripts/MPhys_model.py:159]: yep, that is the one 😊

**Pull request merged**